src/ChipCarousel/README.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'ChipCarousel'
+type: 'component'
+components:
+- ChipCarousel
+categories:
+- Content
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+notes: |
+---
+
+The ``ChipCarousel`` component creates a scrollable horizontal block of chips with buttons for navigating to the previous and next set of chips.
+
+## Basic Usage
+
+```jsx live
+() => {
+  const [offset, setOffset] = useState(120);
+  const [offsetType, setOffsetType] = useState('fixed');
+  const [gap, setGap] = useState(3)
+  
+  return (
+    <>
+      {/* start example form block */}
+      <ExamplePropsForm
+        inputs={[
+          {
+            value: offset,
+            setValue: setOffset,
+            range: {
+              min: 0,
+              max: offsetType === 'percentage' ? 100 : 1000,
+              step: offsetType === 'percentage' ? 1 : 50,
+            },
+            name: 'offset'
+          },
+          {
+            value: offsetType,
+            setValue: setOffsetType,
+            options: ['percentage', 'fixed'],
+            name: 'offsetType'
+          },
+          {
+            value: gap,
+            setValue: setGap,
+            range: { min: 0, max: 6, step: 0.5 },
+            name: 'offset'
```

Comment:
[curious] It looks like there's no difference in the gap between `0` and `0.5`; is this expected?

<img width="942" alt="image" src="https://github.com/openedx/paragon/assets/2828721/ba140860-abd7-46b6-96c4-230970142d5e">

<img width="940" alt="image" src="https://github.com/openedx/paragon/assets/2828721/803199ab-03e6-4975-8a8e-2dcf1dc555c7">

Reply:
Yes, probably I have talked about this case that there is no `.5` `spacer` value.
```
$spacers: map-merge(
  (
    0: 0,
    1: ($spacer * .25),
    1\.5: ($spacer * .375),
    2: ($spacer * .5),
    2\.5: ($spacer * .75),
    3: $spacer,
    3\.5: ($spacer * 1.25),
    4: ($spacer * 1.5),
    4\.5: ($spacer * 2),
    5: ($spacer * 3),
    5\.5: ($spacer * 4),
    6: ($spacer * 5),
  ),
  $spacers
);
```

Reply:
Ah, yes. Thank you for the reminder. Noted :)